docs/source/doctest_fixtures.py

```diff
@@ -10,6 +10,9 @@
 so all imports and variables are available in any doctest.
 The output of this setup script does not show up in the documentation.
 """
+import logging
+
+logging.basicConfig(level=logging.WARN)
```

Comment:
wait isn't the python default warn? is somewhere else setting it to info?

Reply:
Not sure, but when you run doctests, it prints INFO statements, which clutter up stdout and stderr.